<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Серьёзный пэт №3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Общий объём (в ГБ)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Количество файлов</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLCDNumber" name="lcd_Count">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLCDNumber" name="lcd_Volume">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="smallDecimalPoint">
           <bool>false</bool>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_Dlg_Name">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>20</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Введите текст:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_Dlg_Name">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>60</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::NoWrap</enum>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="overwriteMode">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxCaseSensetive">
        <property name="text">
         <string>Учитывать регистр</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxSearchFullPhrase">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Искать слово(фразу) целиком</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Вложенные директории</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea_Subdirectories">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>1000</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>348</width>
           <height>98</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="label_Lvl1">
            <property name="maximumSize">
             <size>
              <width>350</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_Lvl1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>320</width>
              <height>25</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>350</width>
              <height>0</height>
             </size>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::DefaultContextMenu</enum>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="button_Search">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Пока без понятия зачем эта кнопка</string>
          </property>
          <property name="toolTipDuration">
           <number>10</number>
          </property>
          <property name="text">
           <string>Поиск</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_Clear">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Сброс</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_Stop">
          <property name="text">
           <string>Остановка поиска</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="MyQTreeWidget" name="treeWidget">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="headerCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="headerHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="headerStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">FileName</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">PozInFile</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>FullString</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="OpenDlgChoosePath"/>
    <addaction name="Reload"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="OpenDlgChoosePath">
   <property name="text">
    <string>Основные</string>
   </property>
  </action>
  <action name="Reload">
   <property name="text">
    <string>Перезагрузить</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyQTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>MyObjects/myqtreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
